File: ruby/7_kyu/cw_disemvowel.rb
def disemvowel(str)
  vowels = "aeiou"
  new_str = ""
  str.each_char { |char| new_str += char unless vowels.include?(char.downcase) }
  new_str
end

# https://www.codewars.com/kata/52fba66badcd10859f00097e

File: javascript/7_kyu/cw_disemvowel.js
function disemvowel(str) {
  let newStr = "";
  vowels = "aeiou".split("");
  for (i = 0; i < str.length; i++) {
  if (vowels.includes(str[i].toLowerCase())) {
      newStr += ""
    } else {
      newStr += str[i];
    }
  }
  return newStr;
}

// https://www.codewars.com/kata/disemvowel-trolls/javascript